pkgs/development/python-modules/pyobjc-framework-CalendarStore/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, buildPythonPackage, pythonOlder, fetchPypi, darwin, python,
+pyobjc-core, pyobjc-framework-Cocoa }:
+
+buildPythonPackage rec {
+  pname = "pyobjc-framework-CalendarStore";
+  version = "6.2.2";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "117qqmp944cjcdfz218rb3hx1xni53d781sjx3sdp77gpg2kph0y";
+  };
+  
+  postPatch = ''
+    # Hard code correct SDK version
+    substituteInPlace pyobjc_setup.py \
+      --replace 'os.path.basename(data)[6:-4]' '"${darwin.apple_sdk.sdk.version}"'
+  '';
+
+  buildInputs = [
+  ] ++ (with darwin; [
+  ] ++ (with apple_sdk.frameworks;[
+  ]));
```

Comment:
Should those empty blocks stay there all the time?

Reply:
I can try to remove them and I think I never use the first one anyway.